public/app/features/dashboard-scene/scene/DashboardSceneUrlSync.ts

```diff
@@ -135,11 +138,12 @@ export class DashboardSceneUrlSync implements SceneObjectUrlSyncHandler {
       }
       if (getLibraryPanel(panel)) {
         this._handleLibraryPanel(panel, (p) => {
-          this._scene.setState({ editPanel: buildPanelEditScene(p) });
+          this._scene.setState({ editPanel: buildPanelEditScene(p, isNewPanel) });
         });
```

Comment:
how can we ever have a new library panel when go into panel edit?

Reply:
In the current experience, you can't. You're completely right. I can remove this one.